com.unity.multiplayer.mlapi/Tests/Editor/MessagePacker.cs

```diff
@@ -0,0 +1,29 @@
+using MLAPI.Serialization;
+using MLAPI.Messaging;
+using MLAPI.Serialization.Pooled;
+
+namespace MLAPI.EditorTests
+{
+    internal static class MessagePacker
+    {
+        internal static NetworkBuffer WrapMessage(MessageQueueContainer.MessageType messageType, NetworkBuffer messageBody, bool useBatching)
```

Comment:
ugh, I hope we'd eventually get rid of this re-packing/wrapping soon.

Reply:
FWIW MessagePacker is only used in tests now. That's why I moved it to the Tests directory.

Reply:
oh right! my bad :)